pkg/web/static/js/transactionInfo.js

```diff
@@ -30,15 +30,6 @@ document.body.addEventListener('htmx:afterSettle', (e) => {
     const humanizeLastUpdate = dayjs(lastUpdate.textContent).fromNow();
     lastUpdate.textContent = humanizeLastUpdate;
   }
-
-  // Display envelope timestamp in local time.
-  if (e.detail.requestConfig.path === `/v1/transactions/${transactionID}/secure-envelopes` && e.detail.requestConfig.verb === 'get') {
```

Comment:
This was removed since the time zone wasn't displayed and we would've encountered the bug I discovered about the offset appearing instead of the time zone. Also, the timestamp should be consistent with others in the UI.